loader/icd.h

```diff
@@ -121,6 +121,11 @@ void khrIcdContextPropertiesGetPlatform(
     const cl_context_properties *properties, 
     cl_platform_id *outPlatform);
 
+// translate device type to platform
+void khrIcdDeviceTypeGetPlatform(
```

Comment:
Same comment as with https://github.com/KhronosGroup/OpenCL-ICD-Loader/pull/79: let's include this as part of a separate PR.  It doesn't look like it does anything right now, anyhow.

Reply:
Removed.